olive/passes/pytorch/tensor_parallel.py

```diff
@@ -70,7 +68,7 @@ def _run_for_config(
         # 4. Save the weights for each rank
         for rank in range(world_size):
             self.load_rank_weights(pytorch_model, rank, world_size)
-            output_model_name = PyTorchTensorParallel.RANKED_MODEL_NAME_FORMAT.format(rank)
+            output_model_name = DistributedPyTorchModel.DEFAULT_RANKED_MODEL_NAME_FORMAT.format(rank)
```

Comment:
It is not actually part of this PR but I think we should have done something like
```
try:
    line 62-74
finally:
    line 75-76
```
to ensure that the layers and initializers are reset even when there is an exception?

Reply:
Done!